wordpress-mobile/WordPress-Android
# My Site page for users without a site

Closes #2734. This PR adds a better design for the my site page when the user doesn't have any blogs. Here is a final screenshot of the design: https://cloudup.com/cn8EOTmYXp8

Testing instructions:
* If you don't have one, create a new WordPress.com account without a site. You can use http://wordpress.com/start/user for that.
* Login to the app using the newly created account
* In my sites tab, you should see the new design
* Tap on the "Add Site" button and create a new blog
* You should see the page refreshed and your newly created blog show up

I only have one concern about the implementation. I am hiding the FAB when there are no sites and showing it if there is one. I was wondering if that could affect anything. I am not sure if the FAB is always visible or if there is another mechanism that hides it. Just to be safe I wanted to mention it.
